backend/src/server/createServer.ts

```diff
@@ -75,6 +75,13 @@ const createServer = async (
     logger,
   })
   apollo.applyMiddleware({ app, path: '/' })
+  logger.info(
+    `running in environment PRODUCTION = ${CONFIG.PRODUCTION} with TEST_MODUS = ${CONFIG.TEST_MODUS} ...`,
+  )
+  if (CONFIG.PRODUCTION && CONFIG.TEST_MODUS === 'true') {
```

Comment:
I think this is not going to work. In line 124 of `deployment/bare_metal/start.sh` we have `export NODE_ENV=production`, so on all our servers we will never be able to set the test mode to true

Reply:
i've introduced an environment variable GRADIDO_ENV_NAME, which is handled in the start.sh, documented in the setup.md and exported on stage1 and stage2 yet. 
For local environments it can also be set in the .env by init the NODE_ENV with the required value like development/stage1/stage2/stage3/production